tests/test_resolvers/test_cache.py

```diff
@@ -1,9 +0,0 @@
-# -*- coding: utf-8 -*-
```

Comment:
I found these tests. Somebody committed these and I'm not sure why they still exist!